src/middleware/combine/index.ts

```diff
@@ -51,7 +57,9 @@ export const some = (...middleware: (MiddlewareHandler | Condition)[]): Middlewa
         break
       } catch (error) {
         lastError = error
-        continue
+        if (isNextCalled) {
+          break
+        }
```

Comment:
I think these lines `#L61 - L62` are covered, though Codecov says they are not covered. Should you add more tests?

Reply:
Thanks for your comment.

The refactoring of the e0cf156 test had failed 😢  Fixed in 033d48b.

Reply:
I see!